src/Utils/ErrorMessageBuilder.ts

```diff
@@ -0,0 +1,22 @@
+import { isAxiosError } from 'axios';
+
+export class ErrorMessageBuilder {
+  public static buildMessage(
```

Comment:
It is handling too many responsibilities (assertions, extraction, and formatting); split it into smaller, manageable methods